### DISCUSSION TITLE
Dropwizard 2.1.0 Release Notes

### DISCUSSION THREAD

**Discussion by tylerlevee** [dropwizard/dropwizard#5283] [Q&A] [answered]
Hello,

I noticed the release of Dropwizard 2.1.0 on the Maven repository. Is this the full release/is this safe to use? Will release notes be coming soon?

Thank you.

**Comment by zUniQueX** [comment]
Hi @tylerlevee. The released version is the final release and we are working on release notes. An upgrade guide is already available at https://www.dropwizard.io/en/latest/manual/upgrade-notes/upgrade-notes-2_1_x.html.

**Comment by zUniQueX** [comment, marked as answer]
The release notes are available: https://github.com/dropwizard/dropwizard/releases/tag/v2.1.0
